test/e2e/configuration_anomaly_detection_test.go

```diff
@@ -62,6 +66,10 @@ var _ = Describe("Configuration Anomaly Detection", Ordered, func() {
 
 		provider, err = k8s.GetProvider(ctx)
 		Expect(err).NotTo(HaveOccurred(), "Could not determine provider")
+
+		pdRoutingKey := os.Getenv("CAD_PD_TOKEN")
```

Comment:
Is this supposed to be the following?
```
pdRoutingKey := os.Getenv("PAGERDUTY_ROUTING_KEY")
```

Also, awould these variables already be set up in the e2e pipelines if we merge this now?